modules/ROOT/pages/functions/scalar.adoc

```diff
@@ -1499,7 +1499,7 @@ With this in mind, the below list contains all supported types (as of Neo4j 5.13
 
 This should be taken into account when relying on the output of the `valueType()` function.
 
-See the xref::syntax/expressions.adoc#type-predicate-expressions[type predicate expression] for an alternative way of testing type values.
+See the xref::values-and-types/type-predicate.adoc[type predicate expression] for an alternative way of testing type values.
```

Comment:
This should be plural as well, right? ("type predicate expressions")

Reply:
Well it is just one expression though 🤔 Would we say "see the Count Expression" or "see the Count Expressions"? :)